docs/using_lagoon/lagoon_yml.md

```diff
@@ -171,6 +171,14 @@ In the `"www.example.com"` example repeated below, we see two more options \(als
             hsts: max-age=31536000
 ```
 
+#### Monitoring a specific path
+When [UptimeRobot](https://uptimerobot.com/) is configured for your cluster (OpenShift or Kubernetes), Lagoon will inject annotations to each route/ingress for use by the `stakater/IngressControllerMonitor`. The default action is to monitor the homepage of the route. If you have a specific route to be monitored, this can be overriden by adding a `monitoring_path` to your route specification. A common use is to set up a path for monitoring which bypasses caching to give a more real-time monitoring of your site.
+
+```
+     - "www.example.com":
+            monitoring_path: "/bypass-cache"
```

Comment:
all other yaml keys are separated by dashes, can we instead of `monitoring_path` use `monitoring-path`?

I know that `monitoring_urls` was underline but that was actually already not correct :)